targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_EP_AGORA/ONBOARD_TELIT_ME910.cpp

```diff
@@ -114,7 +114,7 @@ nsapi_error_t ONBOARD_TELIT_ME910::init()
     // AT#PORTCFG=0
     // Set command allows to connect Service Access Points to the external physical ports giving a great
     // flexibility. Examples of Service Access Points: AT Parser Instance #1, #2, #3, etc..
-    _at->at_cmd_discard("#PORTCFG", "=3");
+    _at->at_cmd_discard("#PORTCFG", "=", EP_AGORA_PORT_CONFIGURATION_VARIANT);
 
```

Comment:
This line is missing a "%d".
Should be _at->at_cmd_discard("#PORTCFG", "=", "%d", EP_AGORA_PORT_CONFIGURATION_VARIANT);

Reply:
Ah, you're right! Change added.